models/reports/sql/msf/lab-follow-ups-line-list.sql

```diff
@@ -0,0 +1,37 @@
+select
+    display_id as "{{ translate_label('patientDisplayId', 'Patient ID') }}",
+    first_name as "{{ translate_label('patientFirstName', 'First Name') }}",
+    last_name as "{{ translate_label('patientLastName', 'Last Name') }}",
+    age as "{{ translate_label('patientAge', 'Age') }}",
+    sex as "{{ translate_label('patientSex', 'Sex') }}",
+    conditions as "{{ translate_label('patientNCDConditions', 'NCD') }}",
+    test_name as "{{ translate_label('labTestName', 'Lab test') }}",
+    requested_datetime::date as "{{ translate_label('labTestDate', 'Date of last lab test') }}",
```

Comment:
Should separate the translation label as not all labTestDate will be last lab test.
```suggestion
    requested_datetime::date as "{{ translate_label('labTestLastDate', 'Date of last lab test') }}",
```